packages/big-design/src/components/Radio/Radio.tsx

```diff
@@ -3,11 +3,12 @@ import React, { forwardRef, useMemo, Ref } from 'react';
 import { useUniqueId } from '../../hooks';
 import { typedMemo, warning } from '../../utils';
 
-import { HiddenRadio, RadioContainer, StyledRadio } from './styled';
+import { HiddenRadio, RadioContainer, RadioDescription, RadioLabelContainer, StyledRadio } from './styled';
 import { RadioLabel } from './Label';
 
 interface Props {
   label: React.ReactChild;
+  description?: React.ReactChild;
```

Comment:
We are trying to make our components more restrictive and config-based. Do you think it makes sense to make this just a `string`? 

The downside would be if there is a need for something like a "Learn more" url. In that case, we would need to change the API to allow for a description and an optional url.

Reply:
Yeah, pretty much that's why the reason I'm allowing everything. I understand we want it as restrictive as possible, but sometimes we might want links inside the text.

Reply:
I agree, having links within the text will be nice and something we want to do moving forward. @deini how is the help handled on the other inputs? 

Also should we setup an "internal" component for handling help? Looks like a lot of the logic/styles are shared across components.

Reply:
I agree that we will need links. However, we are trying to go as strict as possible so if we can come up with an API that only allows description string + link would be ideal. 

@valterfatia There is a FormControlDescription component.

Reply:
Cool! In that case we should be able to use it here too 👍

Reply:
The string+link works in a scenario where you always have a link at the end as in "Learn more" but I don't think that's always the case. For example, some checkout configurations like privacy policy requires the Store privacy policy to be enabled, so we link directly to the store privacy policy within Checkout, which is not strictly a learn more link.

Reply:
Can’t we make the link text also configurable as well as the target?

Reply:
is not about the link text and target, it's also about where in the description it's located. Doesn't always necessarily need to be at the end.

<img width="755" alt="Screen Shot 2020-03-10 at 3 19 08 pm" src="https://user-images.githubusercontent.com/1621894/76279383-8a430380-62e2-11ea-9092-3d5e855afec5.png">
<img width="550" alt="Screen Shot 2020-03-10 at 3 19 20 pm" src="https://user-images.githubusercontent.com/1621894/76279388-8ca55d80-62e2-11ea-8f30-494ad8a3c780.png">

Reply:
I understand, we had the same issue with Alerts. After chatting with design we ended up agreeing that links would always be in the same place. It keeps things consistent and we can always reword our description. 

I’m not blocking this PR but from experience we will see some snowflake descriptions.

Reply:
the other thing is that sometimes, you might want a link to potentially execute some custom javascript / react modal? I will refactor so it matches alerts, but IMO having very strict properties tend to causes more problems than solutions, specially for something trivial like a description.

Reply:
Yeah, another scenario is if you need a tooltip somewhere in your description or an icon. That's why we need a bit more guidance on this from design.

Reply:
@valterfatia  any ideas on this? Are you happy for us to restrict radio description to one link max? Or do you foresee us being able to include more than one link, or special behaviours like opening a modal / tooltip?

Reply:
I think it will be fine if we restrict to a single link at the end as long as the text of the link is configurable. That way we can voice it as "Learn more" or as an action like "Go to x".